include/internal/internal.h

```diff
@@ -1,9 +1,35 @@
 #ifndef ERD_INTERNAL_H
 #define ERD_INTERNAL_H
 
+#include <string>
 #include <vector>
 #include <cstdint>
 
+#define KEY_FILE_VERSION 4U
+#define KEY_FILE_DERIVATION_FUNCTION std::string("scrypt")
+#define KEY_FILE_CIPHER_ALGORITHM std::string("aes-128-ctr")
+
 typedef std::vector<uint8_t> bytes;
 
+struct KdfParams
+{
+    uint32_t dklen = 32;
+    uint32_t n = 4096;
+    uint32_t r = 8;
+    uint32_t p = 1;
+    std::string salt;
+};
+
+struct EncryptedData
```

Comment:
Perhaps also add a reference to this URL?

https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/main/src/crypto/encryptedData.ts

Reply:
Added comment